crates/cli/src/validation.rs

```diff
@@ -13,6 +15,19 @@ pub fn is_class_hash_valid(hash: &str) -> bool {
     false
 }
 
+pub fn expand_tilde(path: &String) -> Utf8PathBuf {
```

Comment:
On another note, since we are not really reusing this anywhere else, I believe we shouldn't extract this into its own function. Let's only do this when we are reusing this

Reply:
sure, let's do it.